backend/fms_core/migrations/0041_auto_20220623_1801.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1 on 2022-06-23 18:01
```

Comment:
You might want to include those 2 auto-migrations into one 0040 migration file for simplicity.